.github/copilot-instructions.md

```diff
@@ -0,0 +1,145 @@
+# Ontology of Biological Attributes (OBA) Project Guide
+
+This includes instructions for editing the OBA ontology. 
+
+## Project Layout
+- Main development file is `src/ontology/oba-edit.obo`
+- ODK and ontology documentation can be found in `/docs/`
+
+## Querying ontology
+
+- To look at a specific term if you know the ID:
+    - `obo-grep.pl -r 'id: OBA:2040177' src/ontology/oba-edit.obo`
+- All mentions of an ID
+    - `obo-grep.pl -r 'OBA:2040177' src/ontology/oba-edit.obo`
+- Note that `obo-grep.pl` is in your PATH, no need to search for it    
+- Only search over `src/ontology/oba-edit.obo`
+- DO NOT bother doing your own greps over the file, or looking for other files, unless otherwise asked, you will just waste time.
+- ONLY use the methods above for searching the ontology
```

Comment:
This wont work for oba - most of the ids are in definitions.owl, and in functional syntax.